rdbms-example/pom.xml

```diff
@@ -14,14 +14,6 @@
   <version>1.0.0</version>
   <packaging>jar</packaging>
   
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <version.io.fabric8>3.5.41</version.io.fabric8>
-    <version.postgresql>42.1.4</version.postgresql>
```

Comment:
Can you leave these properties as is, that way if one is copying the example it will work as is. Otherwise they need to grab these from parent pom. That is one of the reasons I did not have a pom.xml when we started